client-src/elements/chromedash-admin-feature-links-page.ts

```diff
@@ -3,7 +3,26 @@ import {showToastMessage} from './utils.js';
 import {SHARED_STYLES} from '../css/shared-css.js';
 import {VARS} from '../css/_vars-css.js';
 import {LAYOUT_CSS} from '../css/_layout-css.js';
+import {customElement, property, state} from 'lit/decorators.js';
 
+interface FeatureLink {
+  url: string;
+  http_error_code?: number;
+  feature_ids: number[];
+}
+
+interface FeatureLinksSummary {
+  total_count: number;
+  covered_count: number;
+  uncovered_count: number;
+  error_count: number;
+  http_error_count: number;
+  link_types: {key: string; count: number}[];
+  uncovered_link_domains: {key: string; count: number}[];
+  error_link_domains: {key: string; count: number}[];
+}
```

Comment:
I think these types belong in cs-client.js, near the operation that returns them. You can do that while it's still a JS file, by copying the (other) `FeatureLink` example. You'll need to pick a different name for the type.

Reply:
pick a different name for the one in cs-client? or this one? could you elaborate more? I think the one in cs-client.js is missing features_ids, so we could not just use that without modifying a bit or introducing a new type

Reply:
and also there's no FeatureLinksSummary

Reply:
You need all 3 types defined, they should all be defined in `cs-client.js` near the functions that return them, and you'll need 3 distinct names for them. I don't have strong opinions on what the names are, although copying the name of the function that returns them could be a good strategy, leading perhaps to `SampleFeatureLink`.

Reply:
would it be ok if we put them in type.d.ts for now, as cs-client is still a javascript file?

Reply:
As I said above, you can define them in `cs-client.js` using the `/** @typedef {Object} ... @property ... */` syntax that's already used to define a FeatureLink type there.

https://github.com/GoogleChrome/chromium-dashboard/blob/5ca89c828aba07197557d49d0ec64d15dfe22242/client-src/js-src/cs-client.js#L17-L23

I'd prefer to put them in the right file now, so that nobody has to remember to go back and clean up `types.d.ts` later.

Reply:
will do